src/c/bus-mqtt.c

```diff
@@ -174,9 +174,26 @@ edgex_bus_t *edgex_bus_create_mqtt (iot_logger_t *lc, const char *svcname, const
   {
     prot = "tcp";
   }
+  if (strcmp(prot, "mqtt") == 0 || strcmp(prot, "tcp") == 0)
```

Comment:
suggest `if (prot == NULL || *prot == '\0' || strcmp(prot, "mqtt") == 0 || strcmp(prot, "tcp") == 0)` and remove lines 173-176

Reply:
updated